lib/dmv.rb

```diff
@@ -1,16 +1,19 @@
 class Dmv
-
+attr_reader :facilities, :service
   def initialize
     @facilities = []
+    @service = service
   end
 
   def add_facility(facility)
     @facilities << facility
   end
 
   def facilities_offering_service(service)
-    @facilities.find do |facility|
-      facility.services.include?(service)
+    select_service = []
+    @facilities.each do |facility|
+      select_service << facility if facility.services.include?(service)
     end
+    select_service
```

Comment:
This works. Nice job using the accumulator. Could also reach for the find_all enumerable to get rid of the conditional and the accumulator.